### PR TITLE
Packaging coq-ceramist.1.0.1

### PR DESCRIPTION
**Pull request by Gopiandcode**
Changelog
----------------------------------------------------------
- Updated to support coq.8.11.0

### CODE REVIEW

**Comment by clarus** [comment]
By the way, Ceramist takes quite a long time to compile, so that I think it could appear as stuck to some users. Is there a specific reason, like one demo example, or is it just big?

File: released/packages/coq-ceramist/coq-ceramist.1.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+dev-repo: "git+https://github.com/certichain/ceramist.git"
+bug-reports: "https://github.com/certichain/ceramist/issues"
+maintainer: "kirang@comp.nus.edu.sg"
+authors: [
+  "Kiran Gopinathan"
+  "Ilya Sergey"
+]
+license: "GPL-3.0-or-later"
+depends: [
+  "coq" {>= "8.11.0" & < "8.11.1"}
```
Comment (anton-trunov) [inline released/packages/coq-ceramist/coq-ceramist.1.0.1/opam:21]: I guess `< 8.11.1` is due to https://github.com/coq/opam-coq-archive/issues/1230?

Comment (Gopiandcode) [reply released/packages/coq-ceramist/coq-ceramist.1.0.1/opam:21]: Yes, I wasn't  able to install coq-finmap when using `coq.8.11.1`

**Comment by Gopiandcode** [comment]
> By the way, Ceramist takes quite a long time to compile, so that I think it could appear as stuck to some users. Is there a specific reason, like one demo example, or is it just big?

I think it is just big - it takes around an hour on my machine to compile.

**Comment by clarus** [comment]
OK!

**Comment by anton-trunov** [comment]
> I think it is just big - it takes around an hour on my machine to compile.

Do you mean that it takes an hour with the dependencies or just `Ceramist` alone?

Because if it's `Ceramist`, then it's a bit strange because the whole mathcomp library which is around 90 kLoC compiles in under an hour.

**Comment by clarus** [comment]
I am merging but trusting as there is a CI timeout. Anyway there will be another test round with https://coq-bench.github.io/

File: released/packages/coq-ceramist/coq-ceramist.1.0.1/opam (hunk 1 above)
Comment (anton-trunov) [reply released/packages/coq-ceramist/coq-ceramist.1.0.1/opam:21]: PR https://github.com/coq/opam-coq-archive/pull/1232 introduces `coq-mathcomp-finmap` 1.4.1 which fixes this issue and it should be available on opam within a few hours.

**Comment by Gopiandcode** [comment]
@anton-trunov, sorry I missed this message, I meant just `Ceramist` alone.

Part of the reason for the time is some inefficiency on my part - the code makes use of some tactics to automate some of the reasoning about nested bigsums which end up performing a proof search that can last for several minutes. I think there is also a factor that comes from the fact that the library deals with very deeply nested summations (20-30 nested sums) that might also play a part.

**Comment by anton-trunov** [comment]
@Gopiandcode I see, thank you for sharing.

**Pull request merged**